lib/fetchers/PortionFetcher.ts

```diff
@@ -54,18 +54,19 @@ export class PortionFetcher {
     tokenInChainId: number,
     tokenInAddress: string,
     tokenOutChainId: number,
-    tokenOutAddress: string
+    tokenOutAddress: string,
+    forcePortion = false
   ): Promise<GetPortionResponse> {
     metrics.putMetric(`PortionFetcherRequest`, 1);
 
     // we check ENABLE_PORTION for every request, so that the update to the lambda env var gets reflected
     // in real time
-    if (!BACKEND_CONTROLLED_ENABLE_PORTION(process.env.ENABLE_PORTION)) {
+    if (!forcePortion && !BACKEND_CONTROLLED_ENABLE_PORTION(process.env.ENABLE_PORTION)) {
```

Comment:
we need unit test coverage for this.

Reply:
there is already a test suite for this so should not b too bad